blogs/11092025.html

```
@@ -60,29 +59,33 @@
-        <a href="/index.html">Home</a>
+        <a href="/index.html" class="sidenav-brand">Home</a>
         <a href="/projects.html">Projects</a>
         <a href="/blog.html">Blog</a>
       </div>
     </nav>
     
-    <main>
+    <main id="main-content">
       <div class="main">
         <h1>Adnauseam</h1>
         <p><a href="https://adnauseam.i"'>Adnauseam</a> is an ad blocker based of <a href="https://ublockorigin.com">uBlock Origin</a> that I recently switched to. It not only blocks ads, but messes with the ad companies as well. It helps obfusute browsing data.</p>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)